### DISCUSSION TITLE
List DAG runs (batch) - order_by fields

### DISCUSSION THREAD

**Discussion by Kaastor** [apache/airflow#16477] [Q&A]
Hi,

I'm developing an application which is connecting to the airflow system via Airflow REST API. I want to sort DagRuns and I'm using **List DAG runs (batch)** with a **"order_by": "end_date"** payload. 

The problem is no matter what the field I'm putting, I always get **{'order_by': ['Unknown field.']}** response. What I'm doing wrong? 

Best,
Przemek

**Comment by ephraimbuddy** [comment]
What version of airflow are you? Order by was recently added

**Reply by Kaastor** [reply]
Thanks for the reply. I'm on 2.1.0 version.

**Reply by ephraimbuddy** [reply]
Ok. It should work there. Please share the URL and complete payload you're sending. Redact the values

**Reply by Kaastor** [reply]
URL: http://0.0.0.0:7070/api/v1/dags/~/dagRuns/list 
Payload: {
  "page_limit":1,
  "dag_ids":["my_pipeline_id"],
  "order_by":"end_date"
}

Other endpoints seem to work fine. Also "page_limit", and "dag_ids" are working fine (without order_by of course).

**Reply by ephraimbuddy** [reply]
I tested the endpoint with exactly what you have above and no issues. Please share the error message that it gives you.

**Reply by Kaastor** [reply]
Well, all I'm receiving is:  **{'order_by': ['Unknown field.']}**.

Is there something more I can check on my end? I have Airflow set up on docker instance, however, I'm just starting to get into this stuff.

**Reply by ephraimbuddy** [reply]
![postman-dagrunsbatch](https://user-images.githubusercontent.com/4122866/122215307-faa3e180-cea2-11eb-9d69-6bd3cbdb62b1.PNG)

**Reply by ephraimbuddy** [reply]
Make sure your spellings are correct and you're sending a POST request to the above URL

**Reply by Kaastor** [reply]
Thanks for your help, I will dig more into it (probably some silly thing on my end...).
